Templating/FilterExtension.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Templating;
+
+use Liip\ImagineBundle\Templating\Helper\FilterHelper;
+
+class FilterExtension extends \Twig_Extension
+{
+    /**
+     * @var FilterHelper
+     */
+    private $helper;
+
+    /**
+     * @param FilterHelper $helper
+     */
+    public function __construct(FilterHelper $helper)
+    {
+        $this->helper = $helper;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return [
+            new \Twig_SimpleFilter('imagine_filter', array($this->helper, 'filter')),
+        ];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
```

Comment:
`Twig_Extension#getName` is deprecated and should be removed.

See https://github.com/twigphp/Twig/blob/2.x/CHANGELOG#L153